apps/aecore/test/aecore_txs_SUITE.erl

```diff
@@ -117,6 +118,34 @@ txs_gc(Config) ->
 
     ok = aecore_suite_utils:check_for_logs([dev1], Config).
 
+
+micro_block_cycle(Config) ->
+    MBC = aec_governance:micro_block_cycle(),
```

Comment:
Since you don't this value I assume it is the default 3 seconds? With fast mining we have (more than) 1 key block per second, how is this ever going to test the micro_block_cycle time?